libs/log/default.go

```diff
@@ -89,9 +89,9 @@ func getLogFields(keyVals ...interface{}) map[string]interface{} {
 		return nil
 	}
 
-	fields := make(map[string]interface{})
+	fields := make(map[string]interface{}, len(keyVals))
 	for i := 0; i < len(keyVals); i += 2 {
-		fields[keyVals[i].(string)] = keyVals[i+1]
+		fields[fmt.Sprintf("%v", keyVals[i])] = keyVals[i+1]
```

Comment:
I think `fmt.Sprint()` rather than `fmt.Sprintf()` is the right call here, but, yeah.